scripts/deploy-machine.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e #u
+shopt -s expand_aliases
+#set -o xtrace
+
+source scripts/testnet.sh
+source scripts/cleanup.sh
+
+# ---- On Deployment -----
+#
+# This is run on pushes to master, or release/**
+#
+# ENV Variables:
+# - GITHUB_REF: ie. `/refs/heads/mybranch`
+# - FUNDING_PRIV_KEY: funding private key, raw
+# - BS_PASSWORD: database password
+
+# Get version from package.json if not already set
+if [ -z "${RELEASE:-}" ]; then
+  RELEASE=$(node -p -e "require('./packages/hoprd/package.json').version")
+fi
+
+HOPRD_API_TOKEN='D3f4ul+!_2021'
```

Comment:
That would overwrite the value you passed in via the env Github Actions value.

Reply:
Indeed, that's the idea, see my previous comment about using a single token reply.